File: src/main/java/core/car/java/Arguments.java
package core.car.java;

import java.beans.ConstructorProperties;

public class Arguments {
    private String message;
    private String email;
    private int age;



    @Override
    public String toString() {
        return "Arguments{" +
                "message='" + message + '\'' +
                ", email='" + email + '\'' +
                ", age=" + age +
                '}';
    }

    @ConstructorProperties({"message","email","age"})
    public Arguments(String message, String email, int age) {
        this.message = message;
        this.email = email;
        this.age = age;
    }
}

File: src/main/java/core/car/kotlin/Arguments.kt
package core.car.kotlin

import java.beans.ConstructorProperties

class Arguments
    @ConstructorProperties("message","email","age")
    constructor(private val message:String,private val email:String,
               private val age:Int){

    override fun toString(): String {
        return "Arguments(message='$message', email='$email', age=$age)"
    }
}